wear/src/main/java/io/homeassistant/companion/android/home/HomePresenterImpl.kt

```diff
@@ -27,7 +27,7 @@ class HomePresenterImpl @Inject constructor(
             "media_player", "remote", "siren", "switch"
         )
         val supportedDomains = listOf(
-            "input_boolean", "light", "switch", "script", "scene"
+            "input_boolean", "light", "switch", "script", "scene", "lock"
```

Comment:
I wonder if we should add this to the `toggleDomain` list so it shows up as  a `ToggleChip` like it does in lovelace?

Reply:
You're right, will add it and change the order in the when statement